File: android/src/main/kotlin/com/sabiantech/sabian_native_common/channels/methods/NotificationChannelHandler.kt
package com.sabiantech.sabian_native_common.channels.methods


import com.sabiantech.sabian_native_common.channels.methods.notifications.NotificationChannelMethodHandler


class NotificationChannelHandler : AbstractMethodChannelHandler() {
    override fun register() {
        register("postNotification", NotificationChannelMethodHandler())
    }

}

File: ios/Classes/channels/methods/NotificationChannelHandler.swift
//
//  NotificationChannelHandler.swift
//  Runner
//
//  Created by bryosabian on 23/11/2023.
//

import Foundation


class NotificationChannelHandler : AbstractMethodChannelHandler {
    override func register() {
        register("postNotification", NotificationMethodChannelHandler())
    }

}
